tests/serverpod_test_server/test/generated_methods/to_json_for_protocol.dart

```diff
@@ -611,7 +611,8 @@ void main() {
     });
   });
 
-  test('Given an object with a server only field then field is serialized.',
+  test(
+      'Given an object with a server only field then that field is not serialized.',
       () {
```

Comment:
Question: Doesn't it make more sense to use this file for the ´toJson` tests now and then use the other file as the `toJsonForProtocol()` tests?

That way we would have the same behavior for all tests here and we could prove that it is just a method call change but the `allToJson()` behavior is now preserved in the `toJson()` method while the `toJson()` behavior is now preserved in the `toJsonForProtocol()` method.